### PR TITLE
put kubenet/bridge into promiscuous mode so pods can reach them selve…

### PR DESCRIPTION
**Pull request by paulgmiller**
…s through service ips: https://kubernetes.io/docs/tasks/debug-application-cluster/debug-service/#a-pod-fails-to-reach-itself-via-the-service-ip

**Reason for Change**:

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [ ] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by devigned** [comment]
@paulgmiller the build is failing due to the generated code being out of date. You should be able to run `make generate` and then commit the changes to get the build to pass.

**Comment by paulgmiller** [comment]
Tried to run from the ferry I'm on right now :)

File: parts/k8s/cloud-init/nodecustomdata.yml [hunk 1]
```diff
@@ -258,6 +258,7 @@ write_files:
             "isGateway": true,
             "ipMasq": false,
             "hairpinMode": false,
+            "promiscMode": true,
```
Comment (jackfrancis) [inline parts/k8s/cloud-init/nodecustomdata.yml:261]: @khenidak @aramase are there any side effects of this change?

Also should we do the equivalent change for Windows + kubenet (though it's not really a recommended configuration)

cc @marosset @jsturtevant

**Comment by bridgetkromhout** [comment]
Hi, @paulgmiller - after discussing with @jackfrancis I'm closing this PR without merging as we've deprecated AKS Engine. Please let us know if there's anything you'd like to do in CAPZ.

**Pull request closed without merging**